src/components/Block/Block.tsx

```diff
@@ -119,39 +120,71 @@ export default observer(function Block(props: BlockProps) {
     }
 
     return (
-        <div 
-            className={`ReactTimeline__Block ${props.className} ${block.selected ? 'ReactTimeline__Block--selected' : ''}`}
-            style={style}
-            draggable="false"
-            onMouseUp={onMouseUp}
-        >
-            {showResizeHandle && (
-                <div 
-                    className="ReactTimeline__Block-handle" 
-                    onMouseDown={e => onResize(e, 'start')} 
-                    style={handleWidth} 
-                />
-            )}
-
-            <div className="ReactTimeline__Block-content" onMouseDown={e => onMouseDown(e)} />
-
-            {showResizeHandle && (
-                <div 
-                    className="ReactTimeline__Block-handle" 
-                    onMouseDown={e => onResize(e, 'end')} 
-                    style={handleWidth} 
-                />
-            )}
-
+        <>
+            {(blocks.groupBy && block.visible) ? <> {
+                block.groupName &&
```

Comment:
can we say `blocks.groupBy && block.visible &&  block.groupName ? ... : ... 
i mean, flattening the checks here